website/source/docs/guides/managing-acl-policies.html.md

```diff
@@ -0,0 +1,150 @@
+
+
+| Rule              | Summary
+--------------------|--------------------------------------------------------
+|`acl`              |ACL rules grant privileges for ACL operations including to create,update, or view tokens and policies.
+|`node` and  `node_prefix` |Node rules grant privileges for node-level registration, including adding agents to the datacenter and catalog.
+| `service` and `service_prefix`   | Service rules grant privileges for service-level registration, including adding services to the catalog. 
+| `operator`        | The operator grants privileges for datacenter operations, including interacting with Raft. 
+
+
+On the API pages, each endpoint will have a table that includes required ACLs. The node health endpoint, shown below, requires node and service read to view all checks for the specified node. 
+
+![API Health Endpoint](/static/img/consul/api-endpoint.png 'Screenshot of Health endpoint page')
+
+Finally, before using a token in production, you should test that it has the correct privileges. If the token being used is missing a privilege, then Consul will issue a 403 permission denied error. 
+
+```sh
+$ consul operator raft list-peers
+Error getting peers: Failed to retrieve raft configuration: Unexpected response code: 403 (rpc error making call: Permission denied)
+``` 
+
+### Operations Example: Listing Consul Agents
+
+To view all Consul agents in the datacenter, you can use the `consul members` CLI command. You will need at least read privileges for every agent in the datacenter. Depending on your threat model you could have either of the following policies. 
+
+An individual rule and policy for each agent: this requires you to add a new policy to the token every time a new agent is added to the datacenter. Your token would have as many policies as agents in the datacenter. This method is ideal for a static environment, and for maximum security.
+
+```hcl
+agent "server-one" 
+  { policy = "read"
+  }
+```
+
+If you have a dynamic or very large environment, you may want to consider creating one policy that can apply to all agents, to reduce the Operation Team’s workload. 
+
+```hcl
+agent_prefix "" 
+  { policy = "read"
+  }
+```
+
+## Secure Access Control: Operator-Only Access
+
+The most secure access control implementation restricts tokens with `acl=”write”` policies to only one or a few trusted operators. Tokens with the policy `acl = "write"` grant the holder unlimited privileges, because they can generate tokens with any other resource and policy. The operators are responsible for creating all other policies and tokens that grant limited access to the datacenter. We refer to this implementation as the operator-only implementation. This implementation type is the most secure, and most complex to manage.  
+
+For this implementation type operators are responsible for managing policies and tokens for: 
+- service registration
+- Connect proxy registration
```

Comment:
I'm in two minds about whether it's more or less clear to call this out. Technically proxy registration is also just service registration but for a proxy type of service. But it may not be obvious to users that that is the case? The risk of calling them out separately is that it might imply that there is a different process in terms of ACL for these two when there kind of isn't.

I'm fine leaving it if you think it's clearer as I'm super on the fence!

Reply:
I'm putting them on the same line! My original thought was sometimes users will register the proxy and service separately. Also, even though the registration process is the same users still need to actively think about registering proxies since its not automatic.